accept-credential-component.js

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Accept Credential directive.
+ *
+ * Copyright (c) 2015-2016 Digital Bazaar, Inc. All rights reserved.
+ *
+ * @author Dave Longley
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brAcceptCredential', {
+    bindings: {
+      onComplete: '&brAcceptCredentialCallback',
```

Comment:
I'm mostly going for something that uses a consistent naming convention: `brOnSomething`.